src/components/Modal/ModalCloseButton.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { fireEvent, render } from '@testing-library/react'
+import { LocaleContext } from '../../i18n'
+import ptBr from '../../i18n/locales/pt-BR'
+import { ModalCloseButton } from './ModalCloseButton'
+
+it('should render correctly', () => {
+  const { container } = render(<ModalCloseButton onClick={jest.fn()} />)
+
+  expect(container).toMatchSnapshot()
```

Comment:
I think you can use an inline snapshot here. It's better to keep the snapshot inline for clarity.